packages/ckeditor5-image/src/image/utils.js

```diff
@@ -65,7 +65,7 @@ export function getSelectedImageWidgetOrAncestor( selection ) {
  * @param {module:engine/view/selection~Selection|module:engine/view/documentselection~DocumentSelection} selection
  * @returns {module:engine/view/element~Element|null}
  */
-export function getSelectedImageWidgetAncestor( selection ) {
+export function getSelectionAncestorImageWidget( selection ) {
```

Comment:
Looks like this helper is only used internally by `getSelectedOrAncestorImageWidget()`. I think we should merge them.